docs/NOTICES.md

```diff
@@ -35,6 +35,7 @@
 | `location_without_parent_station`                      	| [`LocationWithoutParentStationNotice`](#LocationWithoutParentStationNotice)                                     	|
 | `missing_calendar_and_calendar_date_files`             	| [`MissingCalendarAndCalendarDateFilesNotice`](#MissingCalendarAndCalendarDateFilesNotice)                       	|
 | `missing_level_id`                         	            | [`MissingLevelIdNotice`](#MissingLevelIdNotice)                                       	                        |
+| `missing_recommended_file`                                	| [`MissingRecommendedFileNotice`](#MissingRecommendedFileNotice)                                                       
```

Comment:
Similar to my comment in RULES.md, we are missing the part at the bottom of the document, where we give the user a table showing what is included in the JSON schema for this notice.

Reply:
⚠️ @KClough We are deprecating NOTICES.md in PR #1132